src/data/data_cleanning_with_outliers.py

```diff
@@ -0,0 +1,208 @@
+#Limpeza de dados a partir do trabalho realizado no N67
+
+import pandas as pd
+import warnings
+warnings.filterwarnings('ignore')
+import os, sys
+
+df = pd.read_csv("../data/raw/train.csv",sep=';', decimal = ',')
```

Comment:
You don't need to change the sep and decimal. Maybe you changed the csv file in your computer.